### DISCUSSION TITLE
Easy way to set up Word Keys on a pre-existing form in Document Template

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1374] [Q&A / Help] [answered]
Ok, I have several forms that we need to complete for various reasons. Similar to the PetPlan.html that is in the system, there are fields in different locations on the page where I need to place the WordKey Tags.

The way the Pet Plan template has been set up is with code placing the WordKey in the exact location on the page - 
<div style="position: absolute; left: 197px; top: 517px; width: 130px; height: 14px; border: 0;">&lt;&lt;DisplayXIfMale&gt;&gt;</div>

This is very time consuming and I just wondered if there is an easier way to do this task?

Once again, thanks in advance! Your assistance is so greatly appreciated :)

**Comment by Tetzor** [comment, marked as answer]
Hi, 

I'm sorry to say there isn't an easier way unfortunately.     

Kind regards,

Jon
